src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,55 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private static final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
     public static void main(String[] args) {
-        // test your code here
+        Manufacturer audi = new Manufacturer();
+        audi.setName("Audi");
+        audi.setCountry("Germany");
+        Manufacturer ferrari = new Manufacturer();
+        ferrari.setName("Fiat");
+        ferrari.setCountry("Italia");
+        System.out.println(manufacturerService.create(audi));
+        System.out.println(manufacturerService.create(ferrari));
+
+        ferrari.setName("Ferrari");
+        System.out.println(manufacturerService.update(ferrari));
+        manufacturerService.delete(ferrari.getId());
+        System.out.println(manufacturerService.getAll());
+
+        List<Driver> allDrivers = driverService.getAll();
+        allDrivers.forEach(System.out::println);
```

Comment:
does it make sense to display an empty list on the screen? because you must have an empty database at the beginning of the program